src/translations/sv_SE.json

```diff
@@ -66,7 +66,7 @@
   "CHECKOUT_EMAIL_PLACEHOLDER": "din.epost@här.se",
   "CHECKOUT_SUMMARY_TITLE": "Dina detaljer",
   "CHECKOUT_SIGN_BUTTON_TEXT": "Signera med mobilt BankID",
-  "CHECKOUT_SIGN_DISCLAIMER": "Genom att trycka på ”Signera med BankId” godkänner jag att jag har tagit del av [förköpsinformation]({PREBUY_LINK}), [villkor]({TERMS_LINK}) och att mina personuppgifter [behandlas enligt GDPR](https:\/\/www.hedvig.com\/privacy).",
+  "CHECKOUT_SIGN_DISCLAIMER": "Genom att trycka på ”Signera med BankId” godkänner jag att jag har tagit del av [förköpsinformation]({PREBUY_LINK}), [villkor]({TERMS_LINK}) och att mina personuppgifter [behandlas enligt GDPR](https:\/\/www.hedvig.com\/se\/personuppgifter).",
```

Comment:
out of scope for this PR but if we updated the landing url did we add a redirect?  :)

Reply:
I'm not sure I understand what you mean.

Reply:
In the market web, when we move pages (such as https://www.hedvig.com/se/privacy -> https://www.hedvig.com/se/personuppgifter), we should add a redirect so that doesn't link to the 404. I understand the move is not made by you but i think it's worth highlighting anyways as this is important to get right for SEO reasons :)

Reply:
Ah, no we don't have any redirects. And `/se/privacy` has probably not been working for a long time. But yes you're right, we should have that. I think this way of working with links is really shaky overall and we should probably not even have it like this, in a text key. But I should wanted to change it quickly so that this link actually leads to an existing page.